src/types/MoujiRecording.ts

```diff
@@ -0,0 +1,6 @@
+import { CursorLocation } from "./CursorLocation";
+
+export type MoujiRecording = {
+  data: Array<CursorLocation>;
+  meta: { interval: number };
```

Comment:
every day we stray further from dogs light

Reply:
:dog: